tools/acorn-optimizer.js

```diff
@@ -790,7 +790,7 @@ function emitDCEGraph(ast) {
       const name = node.id.name;
       nameToGraphName[name] = getGraphName(name, 'defun');
       emptyOut(node); // ignore this in the second pass; we scan defuns separately
-    } else if (node.type === 'FunctionExpression') {
+    } else if (node.type === 'ArrowFunctionExpression') {
```

Comment:
Interesting, I guess we don't optimize the previous form any more with this, but that makes sense.